src/webview/css/css.js

```diff
@@ -32,13 +32,22 @@ code {
   white-space: pre-wrap;
   padding: 0 0.25em;
 }
+code::before {
+  /* Add a left-to-right embedding character before each code block */
+  content: '\\202a';
+}
+code::after {
+  /* And add a pop directional formatting character after each code block */
+  content: '\\202c';
+}
 pre {
   padding: 0.5em;
   margin: 0.5em 0;
   font-size: 0.75em;
   white-space: pre;
   overflow-x: auto;
   word-wrap: normal;
+  direction: ltr;
```

Comment:
The default value of `direction` is `ltr`. Do you need to set this?

Reply:
Yes. When the direction of the message is switched, we don't want the `code` and `pre` tags to be rtl, we want them to stay as they are. So I added `direction: ltr;` to `pre`, and did a more complicated trick for `code` tags (just above this rule).

Reply:
I did test the code without this line and indeed we don't seem to be needing it.
The default is `ltr` so if you do not need to overwrite an already set `rtl` you don't need this line.